app/controllers/home_controller.rb

```diff
@@ -7,15 +7,17 @@ def show
     partial_dir = Rails.root.join("app", "views", "home")
     if user_signed_in?
       redirect_to stream_path
-    elsif is_mobile_device?
+    elsif is_mobile_device? || request.format == :mobile
```

Comment:
No, it's not included in that method: https://github.com/brendanlim/mobile-fu/blob/master/lib/mobile_fu.rb#L89.

@jhass currently it's not possible to manually switch to the desktop version when you are using a mobile device. (because of `is_mobile_device?`) I'd like to remove that method call and just check `request.format == :mobile`. I tested the changes with the FirefoxOS simulator and everything worked like I planned. Do you know if we set the `request.format` somewhere if it is unset so it would be safe to remove the method call?